.github/workflows/example-14.yml

```
@@ -0,0 +1,60 @@
+name: "Example 14: Remove defaults channel"
+
+on:
+  pull_request:
+    branches:
+      - "*"
+  push:
+    branches:
+      - "develop"
+      - "main"
+      - "master"
+  schedule:
+    # Note that cronjobs run on master/main by default
+    - cron: "0 0 * * *"
+
+concurrency:
+  group:
+    ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  example-14:
+    # prevent cronjobs from running on forks
+    if:
+      (github.event_name == 'schedule' && github.repository ==
+      'conda-incubator/setup-miniconda') || (github.event_name != 'schedule')
+    name: Ex14 (${{ matrix.os }}, ${{ matrix.conda-remove-defaults }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["ubuntu-latest", "macos-latest", "windows-latest"]
+        conda-remove-defaults: ["true", "false"]
```

How about something like

```suggestion
        explicitly-listed-channels-only: ["true", "false"]
```

Admittedly verby, but I think we can write this in a way that gets closer to the long-term intended behavior

---

See https://github.com/conda-incubator/setup-miniconda/pull/367#pullrequestreview-2347356646. I don't particularly care about the name itself, given it'll be probably be obsolete by March next year.

---

I think the long-term future for this feature is only to migrate conda users to the same pattern as mamba and pixi follow, setting a default channel in the distribution/bootstrapper layer and not hard-coding it in the codebase. That's why I think the name of the parameter for setup-miniconda is okay to focus on conda only, and not required to be named generically.